server/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
```

Comment:
```suggestion
require('dotenv').config();
```
Um env variablen aus der `.env` zu laden